### PR TITLE
Azure > Storage > Storage Account > Minimum TLS Version

### PR DESCRIPTION
**Pull request by judell**

### CODE REVIEW

File: policy_packs/azure/storage/enforce_secure_tls_version_for_storage_accounts/policies.tf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Azure > Storage > Storage Account > Minimum TLS Version
+resource "turbot_policy_setting" "azure_storage_account_minimum_tls_version" {
+  resource = turbot_policy_pack.main.id
+  type     = "tmod:@turbot/azure-storage#/policy/types/storageAccountMinimumTlsVersion"
+  value    = "Check: TLS 1.2"
+  # Uncomment the following line to enforce the policy
```
Comment (SumitPopat) [inline policy_packs/azure/storage/enforce_secure_tls_version_for_storage_accounts/policies.tf:6]:
```suggestion
```

File: policy_packs/azure/storage/enforce_secure_tls_version_for_storage_accounts/main.tf [hunk 2]
```diff
@@ -0,0 +1,5 @@
+resource "turbot_policy_pack" "main" {
+  title       = "Enforce Secure TLS Version for Azure Storage Accounts"
+  description = "Ensure data is protected by using strong encryption protocols, reducing the risk of vulnerabilities associated with older TLS versions"
```
Comment (SumitPopat) [inline policy_packs/azure/storage/enforce_secure_tls_version_for_storage_accounts/main.tf:3]:
```suggestion
  description = "Ensure data is protected by using strong encryption protocols, reducing the risk of vulnerabilities associated with older TLS versions."
```

File: policy_packs/azure/storage/enforce_secure_tls_version_for_storage_accounts/README.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
+---
+categories: ["data protection", "security", "storage"]
+primary_category: "data protection"
+---
+
+# Enforce Azure Storage Accounts Use Minimum TLS Version
```
Comment (SumitPopat) [inline policy_packs/azure/storage/enforce_secure_tls_version_for_storage_accounts/README.md:6]:
```suggestion
# Enforce Secure TLS Version for Azure Storage Accounts
```

File: policy_packs/azure/storage/enforce_secure_tls_version_for_storage_accounts/README.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+Enforce Azure Storage Accounts Use Minimum TLS Version is critical to secure communication between clients and storage accounts. This measure protects data by using strong encryption protocols, reducing the risk of vulnerabilities associated with older TLS versions, and ensuring compliance with security best practices and regulatory requirements.
```
Comment (SumitPopat) [inline policy_packs/azure/storage/enforce_secure_tls_version_for_storage_accounts/README.md:8]:
```suggestion
Enforcing secure TLS version for storage accounts is critical to secure communication between clients and storage accounts. This measure protects data by using strong encryption protocols, reducing the risk of vulnerabilities associated with older TLS versions, and ensuring compliance with security best practices and regulatory requirements.
```

**Review by SumitPopat — CHANGES_REQUESTED** [review]

**Comment by judell** [comment]
image

**Pull request merged**